### PR TITLE
Adding note to performance pages

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes Issue https://github.com/flutter/website/issues/5310

It would be nice to make these pages more cohesive, but this should fix the issue at hand.

Staged:  https://sz-flutter.firebaseapp.com/docs/perf/rendering

### CODE REVIEW

**Comment by sfshaza2** [comment]
@miquelbeltran, this is new. The build is failing on null check again.

**Comment by miquelbeltran** [comment]
PR here: https://github.com/flutter/website/pull/5463

**Comment by sfshaza2** [comment]
Thanks, @miquelbeltran!

File: src/docs/perf/rendering/best-practices.md [hunk 1]
```diff
@@ -4,10 +4,13 @@ short-title: Best practices
...
 
+{% include performance.md %}
+
 Generally, Flutter applications are performant by default,
-so you only need to avoid common pitfalls to get excellent performance
-instead of needing to micro-optimize with complicated profiling tools.
-These best recommendations will help you write the most performant
+so you only need to avoid common pitfalls to get excellent
+performance instead of needing to micro-optimize with
+complicated profiling tools.  These best recommendations
```
Comment (kwalrath) [inline src/docs/perf/rendering/best-practices.md:12]: I'm curious why you aren't breaking after sentences... We should talk.

**Review by kwalrath — APPROVED** [review]

**Comment by sfshaza2** [comment]
I'm definitely ok with that.  :)

**Comment by sfshaza2** [comment]
@miquelbeltran, would you mind taking a look at this? I've merged this with the latest master, and yet the build fails. It complains about apply `dynamic` to a `Widget`.

**Comment by kwalrath** [comment]
> @miquelbeltran, would you mind taking a look at this? I've merged this with the latest master, and yet the build fails. It complains about apply `dynamic` to a `Widget`.

It looks like #5416 broke the build.

**Comment by sfshaza2** [comment]
The build has been fixed in another PR. Forcing a merge.

**Pull request merged**